custom-connectors/Staffbase/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2691 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Staffbase",
+    "description": "This connector provides access to your Staffbase instance",
+    "version": "1.0",
+    "contact": {
+      "name": "Staffbase Support",
+      "url": "https://www.staffbase.com",
+      "email": "support@staffbase.com"
+    }
+  },
+  "host": "powerautomateproxy-main-de1.staffbase.com",
```

Comment:
@Nm-jens Please make sure that the host URL is a production version of your service/API